main.js

```diff
@@ -1,6 +1,42 @@
-// wait for DOM to load before running JS
-$(function() {
-  
-  // your code here
+//way for the document to be ready before executing anything
+$(document).ready (function(){
+	//add onclick handler to all boxes
+	$(".box").on("click",function(){
+		alert("play here");
```

Comment:
This is a really great start. I know it seems small, but most of the logic will happen here when a box is clicked, and you already have the click event working!

Before working on the steps outlined below, let's work on getting and changing the text of the one box that was clicked. Look at the first code example in this section of the jQuery docs to figure out how to get the text of an element that was clicked: http://api.jquery.com/on/#direct-and-delegated-events

Try out the example from the docs in your code when a box is clicked (hint: it will probably print out nothing, since there is no text in the boxes!). After you've done that, work on changing the text. See this example: http://stackoverflow.com/questions/6411696/how-to-change-a-text-with-jquery#answer-6411716

First, focus on changing the text to an "X" (don't work on switching turns just yet)